### PR TITLE
PR: Prepare release

### LINKED ISSUE #5: Create a changelog
See loghub

### LINKED ISSUE #7: Create RELEASE.md instructions

### LINKED ISSUE #17: Create PyPI package

### PR DESCRIPTION
**Pull request by goanpeca**
Fixes #5 
Fixes #7 
Fixes #17

### CODE REVIEW

**Comment by goanpeca** [comment]
@danbradham making the release as we will need this for https://github.com/ColinDuquesnoy/QDarkStyleSheet/pull/138

File: RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+      git clean -xfdi
+
+* Update CHANGELOG.md
+
+* Update version in `__init__.py` (set release version, remove 'dev0')
+
+* Commit changes
+
+      git add .
+      git commit -m "Set release version"
```
Comment (ccordoba12) [inline RELEASE.md]: Please change this to

    git commit -m "Release X.X.X"

where `X.X.X` is the released version.

File: RELEASE.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+      python setup.py sdist bdist_wheel
+
+* Upload distributions
+
+      twine upload dist/* -u <username> -p <password>
+
+* Add release tag
+
+      git tag -a vX.X.X -m 'Release version'
```
Comment (ccordoba12) [inline RELEASE.md]: This should be

    git tag -a vX.X.X -m 'Release X.X.X'

File: RELEASE.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+* Update `__init__.py` (add 'dev0' and increment minor)
+
+* Commint changes
+
+      git add .
+      git commit -m "Restore dev version"
```
Comment (ccordoba12) [inline RELEASE.md]: We usually write this message

    git commit -m "Back to work"

but it's up to you if you want to change it or not.

File: setup.py [hunk 4]
```diff
@@ -45,13 +46,14 @@ def get_description():
     version=get_version(),
     description='Compile SCSS files to valid Qt stylesheets.',
     long_description=get_description(),
+    long_description_content_type='text/markdown',
     author='Yann Lanthony',
     author_email='https://github.com/yann-lty',
     maintainer='Dan Bradham',
```
Comment (ccordoba12) [inline setup.py]: I think this should be

    maintainer='Dan Bradham and the Spyder Project contributors'

Comment (danbradham) [reply setup.py]: It may make more sense to switch it to "the Spyder Project Contributors" at this point. #33

Comment (ccordoba12) [reply setup.py]: Ok, no problem with that.

File: setup.py [hunk 5]
```diff
@@ -45,13 +46,14 @@ def get_description():
     version=get_version(),
     description='Compile SCSS files to valid Qt stylesheets.',
     long_description=get_description(),
+    long_description_content_type='text/markdown',
     author='Yann Lanthony',
     author_email='https://github.com/yann-lty',
```
Comment (ccordoba12) [inline setup.py]: @goanpeca, perhaps we should remove this?

**Comment by goanpeca** [comment]
Fixed @ccordoba12

**Pull request merged**